tools/app-toolkit/scripts/build.sh

```diff
@@ -0,0 +1,89 @@
+TS_CONFIG_TEMPLATES_DOT="${REACT_APP_TEMPLATE_DIR}/configTemplates/ts/.[^.]*"
+BUILD_DIR="${PACKAGE_DIR}/build"
+JS_BUILD_DIR="${BUILD_DIR}/react-app-template.js"
+TS_BUILD_DIR="${BUILD_DIR}/react-app-template.ts"
+JS_TAR="${DIST_DIR}/react-app-template.js.tgz"
+TS_TAR="${DIST_DIR}/react-app-template.ts.tgz"
+
+# cleanup & dir setup
+rm -rf "${DIST_DIR}"
+rm -rf "${BUILD_DIR}"
+mkdir "${DIST_DIR}"
+mkdir "${BUILD_DIR}"
+
+# first build main Application Development Toolkit assets
+npm run build-cjs
+
+#######################################################################
+# Build react-app-template for Typescript
+#######################################################################
+
+# copy sources & essential config from application template project
+# this will be used as-is for TypeScript template
+mkdir -p "${TS_BUILD_DIR}/src"
+cp -R "${REACT_APP_TEMPLATE_DIR}/src" "${TS_BUILD_DIR}"
+cp "${REACT_APP_TEMPLATE_DIR}/package.json" "${TS_BUILD_DIR}"
+cp "${REACT_APP_TEMPLATE_DIR}/webpack.config.js" "${TS_BUILD_DIR}"
+cp "${REACT_APP_TEMPLATE_DIR}/.gitignore" "${TS_BUILD_DIR}"
+cp "${REACT_APP_TEMPLATE_DIR}/README.template.md" "${TS_BUILD_DIR}/README.md"
+
+$PREPARE_PACKAGE_JSON remove-gd-stuff "${TS_BUILD_DIR}"
+
+# 'fork-off' the JS template build dir at this point before adding TypeScript specific configs
+cp -R "${TS_BUILD_DIR}" "${JS_BUILD_DIR}"
+
+# copy over the eslint, prettier and jest config files for the TypeScript project
+cp ${TS_CONFIG_TEMPLATES} "${TS_BUILD_DIR}"
+cp ${TS_CONFIG_TEMPLATES_DOT} "${TS_BUILD_DIR}"
+
+# create archive with TypeScript template
+tar -czf "${TS_TAR}" -C "${TS_BUILD_DIR}" .
+
+#######################################################################
+# Build react-app-template for JavaScript
+#######################################################################
+
+$PREPARE_PACKAGE_JSON remove-ts "${JS_BUILD_DIR}"
+
+# copy over the eslint, prettier and jest config files for the JavaScript project
+cp ${JS_CONFIG_TEMPLATES} "${JS_BUILD_DIR}"
+cp ${JS_CONFIG_TEMPLATES_DOT} "${JS_BUILD_DIR}"
+
+# transpile TypeScript files to JavaScript
+$BABEL_BIN --no-babelrc \
+  --config-file "${PACKAGE_DIR}/.babelrc-js" \
+  --extensions .ts,.tsx "${JS_BUILD_DIR}" -d "${JS_BUILD_DIR}"
+
+# remove TypeScript files
+find "${JS_BUILD_DIR}" -type f \( -iname \*.ts -o -iname \*.tsx -o -iname \*.d.js \) -exec rm -rf {} \;
+
+# rename react files to .jsx
+find "${TS_BUILD_DIR}" -type f -iname \*.tsx | cut -c $((${#TS_BUILD_DIR} + 2))- | cut -sd . -f 1 | xargs -I {} mv "${JS_BUILD_DIR}/{}.js" "${JS_BUILD_DIR}/{}.jsx"
+
+# format transpiled files as format was broken during transpile process
+$PRETTIER_BIN --write "${JS_BUILD_DIR}/**/*.{js,jsx}" \
+  --parser typescript \
+  --print-width 110 \
+  --tab-width 4 \
+  --trailing-comma all
+
+# replace TypeScript related content in JavaScript files
+# node -r esm "./src/processJavaScriptFiles.js" "${SCRIPT_PATH}/../build/tar-source"
```

Comment:
Should this be uncommented?

Reply:
The same thing is commented in the plugin toolkit, so I guess it should be safe to remove? What do you think?

Reply:
Thought so :) yes, let's get rid of it